CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+## Contributing
+
+If you want to contribute to a project and make it better, your help is very welcome. Contributing is also a great way to learn more about social coding on Github, new technologies and and their ecosystems and how to make constructive, helpful bug reports, feature requests and the noblest of all contributions: a good, clean pull request.
```

Comment:
I'd like to make this less generic and more tailored to Zookeeper. What do you think about something like:
```suggestion
If you want to contribute to Zookeeper and make it better, your help is very welcome. Contributing is also a great way to learn more about social coding on GitHub, new technologies and and their ecosystems and how to make constructive, helpful bug reports, feature requests and the noblest of all contributions: a good, clean pull request.
```